lightning-kmp-test-fixtures/src/commonMain/kotlin/fr/acinq/lightning/tests/TestConstants.kt

```diff
@@ -99,7 +104,9 @@ object TestConstants {
     }
 
     object Bob {
-        private val seed = ByteVector32("0202020202020202020202020202020202020202020202020202020202020202")
+        private val entropy = Hex.decode("0202020202020202020202020202020202020202020202020202020202020202")
+        val mnemmonics = MnemonicCode.toMnemonics(entropy)
```

Comment:
nit: you missed that one:
```suggestion
        val mnemonics = MnemonicCode.toMnemonics(entropy)
```

Reply:
done in https://github.com/ACINQ/lightning-kmp/pull/263/commits/58f41dfbe2af9ffa0a031f76121e993c8d47f8d0